.github/workflows/verify_docs-quality.yml

```diff
@@ -30,6 +30,6 @@ jobs:
         with:
           # This also contains --config=.github/vale/config.ini ... :/
           files: '${{ steps.generate.outputs.args }}'
-          version: 2.30.0 # TODO: migrate to v3
+          version: latest
```

Comment:
For the reviewer: I went a bit back and forth on what to put here. You can't do like `^3` or so, it has to be exact versions or `latest` specifically.

Putting a fixed version like `3.0.2` means no security updates and patches. Putting `latest` means we may see some future errors when version 4 is released.

In the end I thought this is the least bad option at this point.

Reply:
Yep 👍